Accordion/Responsive_Accordion_01/code/style.css

```diff
@@ -0,0 +1,127 @@
+html {
```

Comment:
Also avoid directly styling html tags. If possible please move this to a css class